src/strategies/routers/SimpleRouter.sol

```diff
@@ -33,6 +34,19 @@ contract SimpleRouter is
     }
   }
 
+  ///@notice router-dependent implementation of the `pull` function
+  ///@return pulled The amount pulled if successful; otherwise, 0.
+  function __pull__(
```

Comment:
This can be defined as the default __pull__(..., ISignatureTransfer.PermitTransferFrom calldata)` implementation. 
This way you don't need to import `ISignatureTransfer` in classes that do not provide an override to this function